linode/databasepostgresqlv2/framework_models.go

```diff
@@ -141,14 +148,34 @@ func (m *Model) Flatten(
 	m.Platform = helper.KeepOrUpdateString(m.Platform, string(db.Platform), preserveKnown)
 	m.Port = helper.KeepOrUpdateInt64(m.Port, int64(db.Port), preserveKnown)
 	m.Region = helper.KeepOrUpdateString(m.Region, db.Region, preserveKnown)
-	m.RootPassword = helper.KeepOrUpdateString(m.RootPassword, creds.Password, preserveKnown)
-	m.RootUsername = helper.KeepOrUpdateString(m.RootUsername, creds.Username, preserveKnown)
 	m.SSLConnection = helper.KeepOrUpdateBool(m.SSLConnection, db.SSLConnection, preserveKnown)
 	m.Status = helper.KeepOrUpdateString(m.Status, string(db.Status), preserveKnown)
+	m.Suspended = helper.KeepOrUpdateBool(m.Suspended, helper.DatabaseStatusIsSuspended(db.Status), preserveKnown)
 	m.Type = helper.KeepOrUpdateString(m.Type, db.Type, preserveKnown)
 	m.Updated = helper.KeepOrUpdateValue(m.Updated, timetypes.NewRFC3339TimePointerValue(db.Updated), preserveKnown)
 	m.Version = helper.KeepOrUpdateString(m.Version, db.Version, preserveKnown)
 
+	// SSL and credentials may be nil if the database is suspended
+	if ssl != nil {
+		m.CACert = helper.KeepOrUpdateString(m.CACert, string(ssl.CACertificate), preserveKnown)
+	} else {
+		// We always enable perserveKnown here because it will otherwise
```

Comment:
Any thoughts on workarounds for this? Unfortunately most of the clean solutions I found require newer TF and proto versions than our minimum supported version